src/MassTransit/DependencyInjection/DependencyInjection/Registration/Sagas/SagaStateMachineRegistration.cs

```diff
@@ -73,9 +76,9 @@ public void Configure(IReceiveEndpointConfigurator configurator, IServiceProvide
             IncludeInConfigureEndpoints = false;
         }
 
-        ISagaDefinition ISagaRegistration.GetDefinition(IServiceProvider provider)
+        ISagaDefinition ISagaRegistration.GetDefinition(IRegistrationContext registrationContext)
```

Comment:
So, in general, if there is no other `context` in scope, I would prefer this just be `context` instead of `registrationContext`. Yes, super minor, but just reduces the string length.